compiler/crates/relay-compiler/src/artifact_content/content.rs

```diff
@@ -39,6 +39,115 @@ use super::content_section::GenericSection;
 use crate::config::Config;
 use crate::config::ProjectConfig;
 
+pub fn generate_preloadable_query_parameters(
+    config: &Config,
+    project_config: &ProjectConfig,
+    printer: &mut Printer<'_>,
+    schema: &SDLSchema,
+    normalization_operation: &OperationDefinition,
+    query_id: &QueryID,
+) -> Result<Vec<u8>, FmtError> {
+    let mut request_parameters = build_request_params(normalization_operation);
+    let cloned_query_id = Some(query_id.clone());
+    request_parameters.id = &cloned_query_id;
+
+    let mut content_sections = ContentSections::default();
+
+    // -- Begin Docblock Section --
+    let v = match query_id {
```

Comment:
nit: Prefer explicit variable names
```suggestion
    let extra_annotations = match query_id {
```